course02/homework01/gavin/list.h

```diff
@@ -0,0 +1,47 @@
+//
+// Created by gavin on 19/12/2018.
+//
+
+#include <cstdio>
+#include <memory>
+#include <iostream>
+struct Element
+{
+    Element(int data) : mData(data)
+    {}
+    int mData{};
```

Comment:
No need for {} here, you don't have a defaulted ctor anymore as there is a user defined one.